README.md

```diff
@@ -143,3 +143,8 @@ Type: `String`
 
 ## Testing Use Cases / Recipes
 Please visit our [Wiki](https://github.com/stsvilik/wdio-docker-service/wiki) for more details.
+
+Run automation scripts by creating GRID + NODE : https://youtu.be/NDt4alzH5E0
```

Comment:
@17thSep while this video is useful, I found very little example of how docker service is used. Second vide has a little more, info but again fails to showcase advantages of this service.

Is it possible for you to do videos which specifically showcase two ways this service can be used:
a) as a way to test a containerized application (custom app in a Docker)
b) as a Selenium host testing a running site

This way it will be more relevant :)

Reply:
ok